R/loadModule.R

```diff
@@ -67,7 +67,9 @@ loadModule <- function( module, what = character(), loadNow,
         loadNow <- !is(loadM, "error")
     }
     if(loadNow) {
-        .botched <- isBotchedSession()
+        ## .botched <- isBotchedSession()
+        .botched <- FALSE
```

Comment:
Do we know why we had this check?

The comment above `isBotchedSession()`:

```
## the following items are to get around some insanity in the
## CMD check of packages using Rcpp that dies in loadModule()
## because some code somewhere can't find the methods package
isBotchedSession <- function()
    ! ("package:methods" %in% search())
```

This, at least to me, implies that there is code within Rcpp modules that requires `methods` not only to be loaded but also attached; do we have some way of verifying that Rcpp modules work when methods is loaded but not attached?